packages/openactive-integration-tests/test/helpers/flow-stages/cancel-order.js

```diff
@@ -92,6 +92,22 @@ class CancelOrderFlowStage extends FlowStage {
   static getFirstOrderItemIdFromB(bFlowStage) {
     return CancelOrderFlowStage.getFirstOrderItemId(() => bFlowStage.getOutput().httpResponse.body);
   }
+
+  /**
+   * 
+   * @param {BFlowStageType} bFlowStage 
+   * @param {number[]} orderItemIndexes 
+   */
+  static getSpecificedOrderItemIdsFromB(bFlowStage, orderItemIndexes) {
```

Comment:
This feels horrible, any suggestions?

Reply:
This is a great point, we shouldn't have any logic that relies on the position of the OrderItem in the array, simply because JSON-LD does not allow for hard guarantees about array ordering.

It's tempting to suggest we randomise the order of the orderedItem array in the reference just to check we haven't accidentally done this anywhere (as the spec doesn't require the ordering to be the same in request and response).

Better to index by the `position`, which is designed for this purpose - B reflects back the `position` provided in the request (https://openactive.io/open-booking-api/EditorsDraft/1.0CR3/#schema-orderitem)

So `orderItemPositions`?